LithoUtils/Classes/Base/Models/CodableUIButtonStyle.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CodableUIButton.swift
+//  LithoUtils
+//
+//  Created by Remmington Damper on 1/9/22.
+//
+
+import Foundation
+import LithoOperators
+import Prelude
+import UIKit
+
+open class CodableUIButtonStyle: CodableViewStyle {
```

Comment:
Let's remove the UI from the titles of these classes. So, for instance, let's make this CodableButtonStyle. Remember to change it both on this line as well as the comment at the top of the class and the name of the file.